src/Wellcome.Dds/Wellcome.Dds.Server/Controllers/PresentationController.cs

```diff
@@ -312,4 +536,12 @@ private string CollectionForAggregationId(string aggregator, string value = null
 
         }
     }
+
+    static class PresentationControllerX
+    {
+        public static string AsV2(this string id)
+        {
+            return id.Replace("/presentation/", "/presentation/v2/");
```

Comment:
We've got this same logic in quite a few places now so probably worth consolidating at some point.